<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatesWidget</class>
 <widget class="QWidget" name="TemplatesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Templates</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="templateWindowLabel">
        <property name="text">
         <string>TemplateWindow</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="tableFilterLabel">
        <property name="text">
         <string>TableFilter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cbTemplateWindow">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editTableFilter"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="viewNameFilterLabel">
        <property name="text">
         <string>ViewNameFilter</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editViewNameFilter"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="nameCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
